interp/handler.go

```diff
@@ -276,7 +276,7 @@ type OpenHandlerFunc func(ctx context.Context, path string, flag int, perm os.Fi
 func DefaultOpenHandler() OpenHandlerFunc {
 	return func(ctx context.Context, path string, flag int, perm os.FileMode) (io.ReadWriteCloser, error) {
 		mc := HandlerCtx(ctx)
-		if !filepath.IsAbs(path) {
+		if path != "" && !filepath.IsAbs(path) {
```

Comment:
Can you add a test for this? for instance, `"" foo`.

Reply:
This is implicitly checked via `test -r ""` (runner: unTest -> open -> openHandler), but that's probably a bit too far away for a clear test.

I've added a test for DefaultOpenHandler.

Reply:
Oh, you probably meant to add just another shell test here rather than a Go test, didn't you. If so I'll respin the patch.

Reply:
Oh, this is the default *open* handler, not the default *exec* handler. My suggestion above about `"" foo` is entirely irrelevant.

My apologies. Please just back out TestDefaultOpenHandler, as the added test cases already cover this code, like you correctly pointed out. More test code isn't necessary.